website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -62,6 +62,18 @@ from the Nomad client by setting [`set_environment_variables`][artifact_env].
 The use of filesystem isolation can be disabled in Client configuration by
 setting [`disable_filesystem_isolation`][artifact_fs_isolation].
 
+#### Server `rejoin_after_leave` (default: `false`) now enforced
+
+All Nomad versions prior to v1.4.4 have incorrectly ignored the Server [`rejoin_after_leave`]
+configuration option. This bug has been fixed in Nomad version v1.4.4.
+
+Previous to v1.4.4 the behavior of Nomad `rejoin_after_leave` was always `true`, regardless of 
+Nomad server configuration, while the documentation incorrectly indicated a default of `false`.
```

Comment:
Gah, sorry my eyes passed right over the version number here. This should be 1.5.0 now too:

```suggestion
All Nomad versions prior to v1.5.0 have incorrectly ignored the Server [`rejoin_after_leave`]
configuration option. This bug has been fixed in Nomad version v1.5.0.

Previous to v1.5.0 the behavior of Nomad `rejoin_after_leave` was always `true`, regardless of 
Nomad server configuration, while the documentation incorrectly indicated a default of `false`.
```

Reply:
Good catch!